File: Workerconnectapp/app/src/main/java/com/example/spider/loginscreen/Model/WorkerTask.java
package com.example.spider.loginscreen.Model;

/**
 * Created by spider on 2/19/2017.
 */
public class WorkerTask {
    public String Description;
    public int WorkerId;
    public int Customer_id;
    public String Status;
    public String Customer_Name;
    public String Ph_No;
    public String Date;
    public int Worker_Task_Id;


    public WorkerTask(String  description, int workerId,int customer_id, String status, String customer_Name, String ph_No
            , String date,int worker_Task_Id) {
        Description = description;
        WorkerId = workerId;
        Customer_id = customer_id;
        Status = status;
        Customer_Name=customer_Name;
        Ph_No=ph_No;
        Date=date;
        Worker_Task_Id=worker_Task_Id;
    }
}

File: webapidemo/webapidemo/Models/WorkerTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace webapidemo.Models
{
    public class WorkerTask
    {
        public String Customer_id { get; set; }

        public String WorkerId { get; set; }

        public String Description { get; set; }
        
        public String Status { get; set; }
    }
}